hpi-schul-cloud/tldraw-client
# Fix zooming cursors and usernames

# Description

QA note:
change tested on env:
 https://bc-6358-zooming-usernames.nbc.dbildungscloud.dev/
No issues found. Size username stays the same when zooming, see tickiet:
https://ticketsystem.dbildungscloud.de/browse/BC-6358



## Links to Tickets or other pull requests



- https://ticketsystem.dbildungscloud.de/browse/BC-6358


## Changes



## Datasecurity



## Deployment



## New Repos, NPM pakages or vendor scripts



## Screenshots of UI changes



## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
